portfolio/src/main/webapp/index.html

```diff
@@ -4,9 +4,10 @@
     <meta charset="UTF-8">
     <title>Nick's Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDuc9LUfW2xFbz9gmQMlQzQbLs-3_f98WA"></script>
```

Comment:
Have you considered using `async` or `defer` for this?   It seems to be [recommended](https://developers.google.com/maps/documentation/javascript/tutorial#Loading_the_Maps_API) and your fellow interns are using them.